lib/beaker-hostgenerator/hypervisor.rb

```diff
@@ -39,12 +39,17 @@ def self.create(node_info, options)
     #                                                and their implementations.
     def self.builtin_hypervisors
       {
-        'vmpooler' => BeakerHostGenerator::Hypervisor::Vmpooler,
-        'vagrant' => BeakerHostGenerator::Hypervisor::Vagrant,
-        'vagrant_libvirt' => BeakerHostGenerator::Hypervisor::Vagrant,
-        'docker' => BeakerHostGenerator::Hypervisor::Docker,
         'abs' => BeakerHostGenerator::Hypervisor::ABS,
+        'container' => BeakerHostGenerator::Hypervisor::Docker,
+        'container_docker' => BeakerHostGenerator::Hypervisor::Docker,
+        'container_podman' => BeakerHostGenerator::Hypervisor::Docker,
+        'container_swarm' => BeakerHostGenerator::Hypervisor::Docker,
+        'docker' => BeakerHostGenerator::Hypervisor::Docker,
+        'docker_podman' => BeakerHostGenerator::Hypervisor::Docker,
```

Comment:
We ended up not having a `Beaker::DockerPodman` in `beaker-docker`, so I guess this is now obsolete?

Reply:
ah indeed